src/Maestro/Maestro.Data/SubscriptionIdGenerator.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Maestro.Data;
+
+public enum RunningService
+{
+    Maestro,
+    PCS
+}
+
+public class SubscriptionIdGenerator(RunningService runningService)
+{
+    private readonly RunningService _runningService = runningService;
+
+    private const string PcsSubscriptionIdPrefix = "00000";
```

Comment:
I checked, there are no subscriptions in BAR that have 00000 as their start

Reply:
Let's replace the first 8 ones